lib/rules/no-descending-specificity/index.js

```diff
@@ -141,11 +142,12 @@ const rule = (primary, secondaryOptions) => {
 function lastCompoundSelectorWithoutPseudoClasses(selectorNode) {
 	const firstChild = selectorNode.nodes[0];
 
-	assert(firstChild);
+	if (!firstChild) return undefined;
```

Comment:
`assert` seemed to be slower than just a falsy check.

It was a minor improvement and I think it would also be ok to keep using `assert` if it makes the code more readable/maintainable.

Reply:
@ybiquitous is best positioned to comment on this one.

Reply:
Agree with replacing `assert` 👍🏼